.github/workflows/constants.yaml

```diff
@@ -0,0 +1,65 @@
+name: Set constants
+
+on:
+  workflow_call:
+    inputs:
+      event_name:
+        required: true
+        type: string
+    outputs:
+      llvm_version:
+        description: "LLVM version"
+        value: ${{ jobs.set-constants.outputs.llvm_version }}
+      mhlo_version:
+        description: "MHLO version"
+        value: ${{ jobs.set-constants.outputs.mhlo_version }}
+      enzyme_version:
+        description: "Enzyme version"
+        value: ${{ jobs.set-constants.outputs.enzyme_version }}
+      python_versions:
+        description: "List of Python versions"
+        value: ${{ jobs.set-constants.outputs.python_versions }}
+      rt_backends:
+        description: "List of backends."
+        value: ${{ jobs.set-constants.outputs.rt_backends }}
+
+# TODO: Commenting this since a deadlock for concurrency group 'Check Catalyst Build-refs/pull/209/merge'
+# was detected between 'top level workflow' and 'constants'
+# concurrency:
+#   group: ${{ github.workflow }}-${{ github.ref }}
+#   cancel-in-progress: true
```

Comment:
You are good to remove this block from this workflow.

Concurrency groups only need to be setup on the calling (or root workflows), not on the called workflows.

In this case since this workflow cannot run on it's own and has to be called by another workflow using workflow_call, there is not need to have a concurrency group.